### PR TITLE
Improvements to DigitalOcean App Platform CLI Example

### PR DESCRIPTION
**Pull request by black-bryan**
Improvements to the 1Password CLI and DigitalOcean command line tool (doctl) deployment of the SCIM bridge to the DigitalOcean App Platform environment for a low cost SCIM bridge.

Improvements include: 

1. Support for Windows (PowerShell) users to perform similar deployments from the ⌘ line (without direct [1Password CLI DigitalOcean Shell plugin support](https://developer.1password.com/docs/cli/shell-plugins/digitalocean)). 
2. Added directions for supporting Google Workspace as the connected IdP to the deployed SCIM bridge.
3. Included resource recommendations for different sized deployments.

### CODE REVIEW

File: beta/do-app-platform-op-cli/README.md [hunk 1]
```diff
@@ -28,31 +37,32 @@ Deploying 1Password SCIM bridge on App Platform comes with a few benefits:
...
+- Access to a terminal on Mac or Linux Bash, Zsh, or on Windows using Powershell
 
 ## Getting started
 
 ### Step 1: Install 1Password and DigitalOcean tools
 
-Install the following on your Mac or Linux machine:
+Install the following on your Mac, Linux or Windows machine:
 
 - 1Password 8 for [Mac](https://1password.com/downloads/mac/) or [Linux](https://1password.com/downloads/linux/)
```
Comment (black-bryan) [inline beta/do-app-platform-op-cli/README.md:48]: Add Windows download link to 1Password 8
```suggestion
- 1Password 8 for [Mac](https://1password.com/downloads/mac/) or [Linux](https://1password.com/downloads/linux/) or [Windows](https://1password.com/downloads/windows/)
```

**Comment by scottisloud** [comment]
Before I provide my review, is this purely a docs update or are there some changes to the deployment itself that you'd like tested out?

**Comment by black-bryan** [comment]
> Before I provide my review, is this purely a docs update or are there some changes to the deployment itself that you'd like tested out?

This is mainly a docs update with alternatives for using Powershell to deploy, with a small update to the deployment yaml when performing GW connection as the IdP. Additional secrets being defined when using the op-scim-bridge-gw.yaml.

File: beta/do-app-platform-op-cli/README.md [hunk 2]
```diff
@@ -28,31 +37,32 @@ Deploying 1Password SCIM bridge on App Platform comes with a few benefits:
   > **Note**
   >
   > If you don't have a DigitalOcean account, you can sign up for a free trial with starting credit: <https://try.digitalocean.com/freetrialoffer/>
-- A Mac or Linux terminal with Bash, Zsh, or Fish
+- Access to a terminal on Mac or Linux Bash, Zsh, or on Windows using Powershell
```
[the remark this answers is not in the document]
Comment (plttn) [reply beta/do-app-platform-op-cli/README.md:40]: Coming back to this, this feels somewhat clunky. 

maybe: "Zsh or Bash on Mac or Linux, or PowerShell on Windows"?

File: beta/do-app-platform-op-cli/README.md [hunk 3]
```diff
@@ -28,31 +37,32 @@ Deploying 1Password SCIM bridge on App Platform comes with a few benefits:
...
 
 ## Getting started
 
 ### Step 1: Install 1Password and DigitalOcean tools
 
-Install the following on your Mac or Linux machine:
+Install the following on your Mac, Linux or Windows machine:
```
Comment (plttn) [inline beta/do-app-platform-op-cli/README.md:46]: Order in style guide order (Mac, Windows, Linux)

File: beta/do-app-platform-op-cli/README.md [hunk 4]
```diff
@@ -28,31 +37,32 @@ Deploying 1Password SCIM bridge on App Platform comes with a few benefits:
...
+- 1Password 8 for [Mac](https://1password.com/downloads/mac/) or [Linux](https://1password.com/downloads/linux/) or [Windows](https://1password.com/downloads/windows/)
 - [1Password CLI 2.9.0](https://developer.1password.com/docs/cli/get-started/#install) or later
 - [`doctl`](https://docs.digitalocean.com/reference/doctl/how-to/install/#step-1-install-doctl)
   > **Note**
   >
-  > **Only** [Step 1: Install doctl](https://docs.digitalocean.com/reference/doctl/how-to/install/#step-1-install-doctl) in the `doctl` installation guide is required for this step.
+  > **Only** [Step 1: Install doctl](https://docs.digitalocean.com/reference/doctl/how-to/install/#step-1-install-doctl) in the `doctl` installation guide is required for this step for Mac and Linux. 
 
-### Step 2: Add your 1Password account and configure the DigitalOcean shell plugin
+### Step 2: Add your 1Password account and configure the DigitalOcean access.
```
Comment (plttn) [inline beta/do-app-platform-op-cli/README.md:55]: Access feels weird? 

"Configure DigitalOcean credentials"?

File: beta/do-app-platform-op-cli/README.md [hunk 5]
```diff
@@ -28,31 +37,32 @@ Deploying 1Password SCIM bridge on App Platform comes with a few benefits:
...
 
 If you haven't already done so, add your 1Password account and connect 1Password CLI to your desktop app, then configure the DigitalOcean shell plugin:
 
-1. [Add your 1Password account](https://support.1password.com/add-account/) to 1Password 8 for Mac or Linux.
+1. [Add your 1Password account](https://support.1password.com/add-account/) to 1Password 8 for Mac or Linux or Windows.
```
Comment (plttn) [inline beta/do-app-platform-op-cli/README.md:59]: ditto, order Mac, Windows, Linux

File: beta/do-app-platform-op-cli/README.md [hunk 6]
```diff
@@ -93,50 +109,204 @@ op item edit "scimsession file" --vault "op-scim" base64_encoded=$(op read "op:/
...
+  ```json
+  {
+      "actor":"admin@example.com",
+      "bridgeAddress":"https://op-scim-bridge-example.ondigitalocean.app"
+  }
+  ```
+
+  Save the file.
+
+  4. Configure the `workspace-settings` for passing to App Platform, just like the `workspace-credentials`, this value needs to the passed to the App Platform as a base64 value. Edit the commands to reference the location/path of the file you saved in the last step.
```
Comment (plttn) [inline beta/do-app-platform-op-cli/README.md:180]: "Needs to be passed"

**Review by plttn — CHANGES_REQUESTED** [review]
Minor nits, but overall looks solid.

File: beta/do-app-platform-op-cli/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (black-bryan) [reply beta/do-app-platform-op-cli/README.md:40]: Applying your suggestion.

```suggestion
- Access to Zsh or Bash on Mac or Linux, or PowerShell on Windows
```

File: beta/do-app-platform-op-cli/README.md (hunk 3 above)
Comment (black-bryan) [reply beta/do-app-platform-op-cli/README.md:46]:
```suggestion
Install the following on your Mac, Windows or Linux machine:
```

File: beta/do-app-platform-op-cli/README.md (hunk 4 above)
Comment (black-bryan) [reply beta/do-app-platform-op-cli/README.md:55]:
```suggestion
### Step 2: Add your 1Password account and configure your DigitalOcean credentials.
```

Comment (black-bryan) [reply beta/do-app-platform-op-cli/README.md:55]:
```suggestion
### Step 2: Add your 1Password account and configure your DigitalOcean credentials.
```

File: beta/do-app-platform-op-cli/README.md (hunk 5 above)
Comment (black-bryan) [reply beta/do-app-platform-op-cli/README.md:59]:
```suggestion
1. [Add your 1Password account](https://support.1password.com/add-account/) to 1Password 8 for Mac, Windows or Linux.
```

File: beta/do-app-platform-op-cli/README.md (hunk 6 above)
Comment (black-bryan) [reply beta/do-app-platform-op-cli/README.md:180]:
```suggestion
  4. Configure the `workspace-settings` for passing to App Platform, just like the `workspace-credentials`, this value is passed to the App Platform as a base64 value. Edit the commands to reference the location/path of the file you saved in the last step.
```

**Pull request merged**